support_policy.md

```diff
@@ -0,0 +1,75 @@
+# Framework Support Policy
+Amazon SageMaker Distribution is a set of Docker images that include popular frameworks for machine learning, data science, and visualization. This page details the framework support policy for SageMaker Distribution Images.
+
+## Support Policy
+The table below outlines the release schedule for SageMaker Distribution Image versions and their planned support timelines. AWS provides ongoing functionality and security updates for supported image versions. In some cases, an image version may need to be designated end of support earlier than originally planned if (a) security issues cannot be addressed while maintaining semantic versioning guidelines or (b) any of our major dependencies, like Python, reach end-of-life. AWS can release ad-hoc major or minor versions on an as-needed basis.
+
+| Version | Description | Release Cadence |
+| :---:   | :---:       | :---:           |
+| Major   | Amazon SageMaker Distribution's major version releases involve upgrading all of its core dependencies to the latest compatible versions. These major releases may also add or remove packages as part of the update. Major versions are denoted by the first number in the version string, such as 1.0, 2.0, or 3.0. | 6 months |
+| Minor   | Amazon SageMaker Distribution's minor version releases include upgrading all of its core dependencies to the latest compatible minor versions within the same major version. SageMaker Distribution can add new packages during a minor version release. Minor versions are denoted by the second number in the version string, for example, 1.1, 1.2, or 2.1. | 1 month |
```

Comment:
As I understand, we are not currently following 1-month, 6-month schedule to release minor/major versions. Once they are added to the policy, does it mean we have to follow this schedule? @balajisankar15 Could you verify if such schedules are valid and should be made public?